web/src/main/resources/application.properties.EXAMPLE

```diff
@@ -1,6 +1,10 @@
 # VEP web API URL with additional parameters. Note that "VARIANT" field will be replaced with
 # the actual variant value before each VEP request.
 vep.url=http://grch37.rest.ensembl.org/vep/human/hgvs/VARIANT?content-type=application/json&xref_refseq=1&ccds=1&canonical=1&domains=1&hgvs=1&numbers=1&protein=1
+
+# VEP web API URL for ids with additional parameters. Note that "VARIANT" field will be replaced with the actual id value before each VEP request.
+vep.url=http://grch37.rest.ensembl.org/vep/human/id/VARIANT?content-type=application/json&xref_refseq=1&ccds=1&canonical=1&domains=1&hgvs=1&numbers=1&protein=1
```

Comment:
@emz1003 this overrides the url specified above ☝️ - i think this shouldn't be part of the myvariant.info PR?